src/networks.json

```diff
@@ -108,6 +108,21 @@
     "explorer": "https://ubiqscan.io",
     "logo": "ipfs://Qmec3HLoN4QhwZAhw4XTi2aN8Wwmcko5hHN22sHARzb9tw"
   },
+  "4181": {
+    "key": "4181",
+    "name": "PHI",
+    "chainId": 4181,
+    "network": "PHI",
+    "multicall": "0xB141139f17c9F6B6B0da82bfC8E1153570AD44fE",
```

Comment:
Is there a way to check the code here?
https://explorer.phi.network/address/0xB141139f17c9F6B6B0da82bfC8E1153570AD44fE

Reply:
Can See Full Documentation[ Here ](https://docs.phi.network)

Reply:
Can find source code of this contract here use the first PR to make any changes https://github.com/snapshot-labs/snapshot.js/pull/591